dali/test/python/operator_2/test_python_function.py

```diff
@@ -341,7 +332,7 @@ def test_python_operator_with_input_sets():
     invalid_pipe = PythonOperatorInputSetsPipeline(
         BATCH_SIZE, NUM_WORKERS, DEVICE_ID, SEED, images_dir, Rotate
     )
-    invalid_pipe.build()
+    invalid_pipe.run()
```

Comment:
Keep build? Error-out at run-time (as opposed to build time) has different semantics (e.g. with external sources), so I guess we should make sure that we fail before running anything if we can.